atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -20,37 +20,35 @@ export const createCrashDataMap = isTempRecord => {
           label: "Crash Date",
           editable: false,
         },
-        crash_time: {
-          label: "Crash Time",
-          editable: false,
-        },
-        day_of_week: {
+        crash_day_of_week: {
           label: "Day of Week",
+          relationshipName: "crashes_list_view",
           editable: false,
         },
         est_comp_cost_crash_based: {
           label: "Est. Comprehensive Cost",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
           format: "dollars",
         },
-        est_econ_cost: {
-          label: "Est. Economic Cost",
-          editable: false,
-          format: "dollars",
-        },
-        speed_mgmt_points: {
-          label: "Speed Management Points",
-          editable: false,
-          format: "text",
-        },
+        // est_econ_cost: {
+        //   label: "Est. Economic Cost",
+        //   editable: false,
+        //   format: "dollars",
+        // },
+        // speed_mgmt_points: {
+        //   label: "Speed Management Points",
+        //   editable: false,
+        //   format: "text",
+        // },
```

Comment:
`speed_mgmt_points` was a custom field we used to calculate in a [trigger](https://github.com/cityofaustin/atd-vz-data/blob/3c4c3df465a079a5a952a45b9b5a3af7a900c5b2/atd-vzd/triggers/atd_txdot_crashes_updates_audit_log.sql#L139). is this something we want to carry over? same for `est_econ_cost`

Reply:
`speed_mgmt_points` is being dropped in the new data model. The old triggers and views that use it will be torn down 🧹

Reply:
what about `est_econ_cost`?

Reply:
it's being dropped! we are only supporting the crash-based cost going forward.